containers-primitive/containers-primitive.cabal

```diff
@@ -23,7 +23,7 @@ library
   exposed-modules:     Map.Prim
   reexported-modules:  Map.Prim as Map
 
-  build-depends:       base, primitive-containers >= 0.3.0
+  build-depends:       base, primitive-containers >= 0.3.0, deepseq
```

Comment:
```suggestion
  build-depends:       base
                     , deepseq ^>= 1.4
                     , primitive-containers ^>= 0.3.0
```

Reply:
Did not know about cabal 2.0's caret operator, great, I applied the diff :)